### ISSUE TITLE
White header for PRs and issues

### ISSUE THREAD

**Issue by Jiri-Kremser** [StylishThemes/GitHub-Dark#800]
this hot-fix worked for me:

```css
.gh-header-hide-if-sticky {
  background-color: #222 !important;
}
```

screenshot before:
![image](https://user-images.githubusercontent.com/535866/49747642-2aefe100-fca4-11e8-85c7-d52ee7821432.png)

screenshot after:
https://i.imgur.com/JHAmczO.png
![image](https://user-images.githubusercontent.com/535866/49747678-40650b00-fca4-11e8-9734-fafcad45bf53.png)

**Comment by Ash258** [comment]
#797, #795, #793, #787, #786 & #785.

**Comment by silverwind** [comment]
This was fixed in 1.20.86, please update.
